Realm/RLMObject.h

```diff
@@ -428,13 +482,41 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ Information about a specific property which changed in an RLMObject change notification.
```

Comment:
It looks like we wrap type names in backticks elsewhere in our documentation comments.